app/assets/stylesheets/course/achievement.scss

```diff
@@ -1,3 +1,17 @@
 // Place all the styles related to the course/achievement controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+$achievement-badge-thumb:   25px;
```

Comment:
This is a global variable (all scss files will see it), so please use the full prefix, $course-achievement-badge-thumb etc.

Maybe we should declare all of this in _variables.scss?